src/resultsdisassemblypage.cpp

```diff
@@ -13,6 +13,7 @@
 #include <QDirIterator>
 #include <QFile>
 #include <QFileDialog>
+#include <QKeyEvent>
```

Comment:
because I cannot comment on the commit message:

please get into the habit of writing better commit messages! a one liner is only acceptable for really trivial typo fixes or similar. a non-trivial patch like this here (and the others) definitely deserves a real commit messages. give a summary of what the patch does (e.g. what kind of keyboard navigation? what keys? for what widget?) also give an explanation for some design choices, i.e. why do you use an even filter instead of `QShortcut` or similar?

better yet, use `KStandardAction`: https://api.kde.org/legacy/pykde-4.7-api/kdeui/KStandardAction.html#findNext https://api.kde.org/legacy/pykde-4.7-api/kdeui/KStandardAction.html#findPrev